### DISCUSSION TITLE
Queue system for playing beatmaps back-to-back

### DISCUSSION THREAD

**Discussion by kstefanowicz** [ppy/osu#29863] [Ideas]
Clone Hero has a "setlist" feature, which allows users to create a list of songs to play in sequence:

[2024-09-13 10-00-02.webm](https://github.com/user-attachments/assets/521cc02f-8b54-4e92-8f43-8e4767c6e252)

I think something similar in Lazer would be pretty cool.

**Comment by yeahirai** [comment]
A mode like this for multi lobbies would be great.

**Reply by bdach** [reply]
...you can already queue up playlist items in multiplayer though?

**Reply by yeahirai** [reply]
OP meant for the selected maps to play like a big marathon. As if it all selected songs were just one big map.

**Comment by Taneta-nemausernames** [comment]
Would even just a next song button at the results screen or a skip results screen setting provide simlar functionality given that you've sorted your maps out at song select?

**Comment by ziv-vy** [comment]
Had a similar idea where the queue would be randomized and infinite (#32911) (wait, I posted my suggestion after yours lol)

And, instead of implementing this as a new feature, just allow playlists beatmaps to be played in succession and allow local playlists to be a thing.
